plugins/modules/content_view.py

```diff
@@ -73,6 +73,11 @@
       - Designate this Content View for importing from upstream servers only.
     type: bool
     version_added: 3.14.0
+  rolling:
+    description:
+      - A rolling content view contains latest synced state of repositories.
+    type: bool
+    version_added: 3.16.0
```

Comment:
```suggestion
    version_added: 5.5.0
```

that's the collection version, not the katello version :)